src/main/resources/stylesheets/snyk_code_suggestion.scss

```diff
@@ -108,3 +108,35 @@ a,
 .high-contrast:not(.high-contrast-light) .light-only {
   display: none;
 }
+
+.example { // AI fix suggestion
+  background-color: var(--container-background-color);
+}
+
+.sn-ai-fixes {
+  padding-top: 20px;
+}
+
+.sn-ai-fix-error {
+  padding-top: 20px;
+}
+
+.sn-fix-wrapper {
+  background-color: var(--container-background-color);  // same variable used in snyk_oss_suggestion.scss
+}
+
+button.generate-ai-fix, button.sn-apply-fix {
+  background-color: var(--generated-ai-fix-button-background-color);
+  color: white;
+}
+
+button.generate-ai-fix:hover, button.sn-apply-fix:hover {
+  opacity: 0.9; /* Slightly reduces opacity on hover */
+}
+
+button.sn-apply-fix {
+  border-radius: 3px;
+  border: 1px solid var(--dark-button-border-default);
+  background: var(--dark-button-default);
+  padding: 4px 12px;
+}
```

Comment:
I wonder if we can just push all of this down to LS, without having to use SCSS at all.

Reply:
Actually, I think even in LS and VSC we shouldn't be using SCSS.
It's much easier to just use CSS and avoid the compilation overhead all together.
Our CSS files are also simple for this use case.

Reply:
I have created jira ticket for this
https://snyksec.atlassian.net/browse/IDE-655